src/main/java/bio/terra/cda/app/util/CDAUtils.java

```diff
@@ -0,0 +1,24 @@
+package bio.terra.cda.app.util;
+
+import java.util.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CDAUtils {
+  private static final Logger logger = LoggerFactory.getLogger(CDAUtils.class);
+
+  /** Split the TABLE.ColumnName into component parts */
+  public static Map<String, String> parseTableName(String tableColumnString) {
+    List<String> parts = new LinkedList<String>(Arrays.asList(tableColumnString.split("\\.")));
+    int columnIndex = parts.size() - 1;
+    String column = parts.get(columnIndex); // last bit is the column
+    parts.remove(columnIndex); // remove the column part
+    String tableName = String.join(".", parts);
```

Comment:
Not a big deal, but I think using `String.lastIndexOf('.')` to do this would be a little simpler. You can extract the column name without having to split and join. Something like https://github.com/CancerDataAggregator/cda-service/blob/master/src/main/java/bio/terra/cda/app/util/QueryTranslator.java#L31